document/FileFormat.mdk

```diff
@@ -898,7 +926,7 @@ the precise version number of the logging component that produced the invalid ou
 
 ### `properties` property { #tool-properties }
 
-A `tool` object may contain a property named `properties`, whose value is property bag.
+A `tool` object may contain a property named `properties` whose value is property bag.
```

Comment:
>is property bag [](start = 70, length = 15)

again 'is a property bag'. search for this pattern, i won't repeat the comment if i see it again. :)